doc/examples/beamer.wiki

```diff
@@ -138,7 +138,7 @@ Wiki2Beamer includes some simple formatting commands: \\[4mm]
 * ''Italic Text:'' enclose text in @\textquotesingle\textquotesingle@
 * @Typewriter:@ enclose text in @
 * !Alerts:! enclose text in !
-* _blue_Colors:_ use \_ notation, see project home page \\[4mm]
+* use \_ notation, see project home page \\[4mm]
```

Comment:
don't understand this change, please elaborate.

Reply:
Ahh,  this was related to my unsuccessful attempt to address https://github.com/wiki2beamer/wiki2beamer/issues/40

Removed it for now will dill with it on another PR